ci-operator/config/kiegroup/kie-cloud-operator/kiegroup-kie-cloud-operator-master.yaml

```diff
@@ -29,6 +29,10 @@ build_root:
     name: release
     namespace: openshift
     tag: golang-1.10
+promotion:
+  cluster: https://api.ci.openshift.org
+  name: origin-v4.0
```

Comment:
Since you aren't part of openshift we'll create a separate name and setup for you.

Your namespace should be `kie-cloud` (I'll create that for you) and you can pick the name for your image stream.